assets/js/amazon-wc-admin.js

```diff
@@ -366,7 +366,7 @@
 			window.onbeforeunload = null;
 		} );
 	}
-	if ( $( 'body' ).hasClass( 'post-type-shop_order' ) ) {
+	if ( $( 'body' ).hasClass( 'post-type-shop_order' ) || $( 'body' ).hasClass( 'woocommerce_page_wc-orders' ) ) {
```

Comment:
Maybe replace it with $( 'body' ).is('.post-type-shop_order','.'woocommerce_page_wc-orders') ?

Reply:
That didnt work for me :(

Reply:
Strange. Ok, leave it as it is.

Good to go.